src/test/java/emissary/util/search/MultiKeywordScannerTest.java

```diff
@@ -0,0 +1,115 @@
+package emissary.util.search;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class MultiKeywordScannerTest {
+
+    private String[] defaultKeywords = {"fox", "dog"};
+    private String defaultData = "the quick brown fox jumped over the lazy dog";
+
+    @Test
+    public void testFindAll() {
+        MultiKeywordScanner multiKeywordScanner = new MultiKeywordScanner();
+        multiKeywordScanner.loadKeywords(defaultKeywords);
+        HitList hits = multiKeywordScanner.findAll(defaultData.getBytes());
+        Assert.assertEquals(2, hits.size());
```

Comment:
This is a personal style preference, but I've found that doing a static import of `org.junit.Assert.assertEquals` and its sibling `assert` methods makes the code a bit cleaner and nicer to look at.

Reply:
I'll make the change and push it up soon